pkgs/by-name/wl/wlink/package.nix

```
@@ -21,12 +22,15 @@ rustPlatform.buildRustPackage rec {
 
   cargoHash = "sha256-Hv+W8yFw6zAKwrV6gf9fWOkR/LFNgAD7WwQsHBqTnPI=";
 
-  nativeBuildInputs = [ pkg-config ];
+  nativeBuildInputs = [
+    pkg-config
+  ]
+  ++ lib.optionals stdenv.hostPlatform.isDarwin [ darwin.DarwinTools ];
 
   buildInputs = [
     libusb1
-    udev
-  ];
+  ]
+  ++ lib.optional stdenv.isLinux udev;
```

Can you use `lib.optionals` and `stdenv.hostPlatform.isLinux` like above for consistency? I tend to prefer `lib.optionals` over `lib.optional`.

```suggestion
  ++ lib.optionals stdenv.hostPlatform.isLinux [ udev ];
```